<launch>
  <node name="grabber" pkg="rosSorter" type="grabber.py"/>
  <node name="popositioningsition" pkg="rosSorter" type="positioning.py">
    <param name="X_acceleration" value="500" />  
    <param name="Y_acceleration" value="200" />
  </node>
  <node name="scanner" pkg="rosSorter" type="scanner.py" cwd="node"/>
  <node name="gcodeInterface" pkg="rosSorter" type="gcodeInterface.py"/>
  <node name="extractCards" pkg="rosSorter" type="extractCards.py"/>
  <node name="solveMemory" pkg="rosSorter" type="solveMemory.py"/>
  
  <node type="raspicam_node" pkg="raspicam_node" name="raspicam_node" output="screen">
    <param name="camera_frame_id" value="raspicam"/>  
    <param name="enable_raw" value="false"/>
    <param name="camera_id" value="0"/> 
    <param name="camera_name" value="camerav2_1640x1232"/>
    <param name="width" value="1640"/>
    <param name="height" value="1232"/>

    <param name="exposure_mode" value="antishake"/>
    <param name="awb_mode" value="flash"/>
    <param name="brightness" value="55"/>
    <param name="exposure_compensation" value="5"/>
    <param name="shutter_speed" value="10000"/>
    <param name="framerate" value="10"/>

  </node>
</launch>
